features/reports/reports.feature

```diff
@@ -204,4 +228,20 @@ Feature: Correctly formatted reports
     Examples:
       | option           |
       | -U               |
-      | --ultra-verbose  |
+      | --wiki-links  |
+
+  Scenario Outline: --wiki-links is independent of --line-numbers
```

Comment:
A little confused here - why is this scenario in here? Seems arbitrary to test a random coupling of options.